YoutubeDownloader/ViewModels/Framework/SnackbarService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Avalonia.Threading;
+using Material.Styles.Controls;
+using Material.Styles.Models;
+
+namespace YoutubeDownloader.ViewModels.Framework;
+
+public class SnackbarService
+{
+    private readonly TimeSpan _defaultDuration;
+
+    public SnackbarService(TimeSpan defaultDuration)
+    {
+        _defaultDuration = defaultDuration;
+    }
+
+    /// <summary>
+    /// Posts to the default SnackBarHost
+    /// </summary>
+    public void PostDefault(string message, TimeSpan? duration = null)
```

Comment:
Probably makes sense to just call the methods `Post` since there is no non-default version